contract/src/main/kotlin/org/testing/goody/contracts/Goody.kt

```diff
@@ -0,0 +1,52 @@
+package org.testing.goody.contracts
```

Comment:
testing.org is a real organisation. Try using com.example instead "example.com" is reserved by IANA.